src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
+
 export interface Figure {
+  color: Color,
+  shape: Shape,
 
+  getArea(): number,
 }
 
-export class Triangle {
-
+function isCompare(a: number, b: number, c: number): boolean {
+  return a + b <= c || b + c <= a || c + a <= b;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape = 'triangle' as Shape;
+
+  color: Color;
+
+  constructor(
+    color: Color,
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    this.color = color;
 
+    const isBigger = isCompare(this.a, this.b, this.c);
+
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0 || isBigger) {
+      throw new Error('Error -:)');
+    }
+  }
+
+  getArea(): number {
+    const s: number = (this.a + this.b + this.c) / 2;
+
+    return Math.round(Math.sqrt(s * (s - this.a)
+      * (s - this.b) * (s - this.c)) * 100) / 100;
+  }
 }
 
-export class Rectangle {
+export class Circle implements Figure {
+  shape = 'circle' as Shape;
+
+  color: Color;
 
+  constructor(
+    color: Color,
+    private radius: number,
+  ) {
+    this.color = color;
+
+    if (this.radius <= 0) {
+      throw new Error('Error -:)');
+    }
+  }
+
+  getArea(): number {
+    return Math.floor(this.radius * this.radius * Math.PI * 100) / 100;
```

Comment:
Create variable for area value
Also, remember about DRY
You are using next code `Math.floor(... * 100) / 100;` at least twice, create function for this